pxtsim/langsupport.ts

```diff
@@ -91,6 +91,23 @@ namespace pxsim {
             if (o === null) return "null";
             if (o === undefined) return "undefined;"
             if (o.vtable && o.vtable.name) return o.vtable.name;
+            if (o instanceof RefCollection && (o as any).data) {
+                if ((o as any).data.length > 0) {
+                    let arrayType = typeof (o as any).data[0]
+                    switch (arrayType) {
+                        case "string":
+                        case "number":
+                        case "boolean":
+                            let contents = (o as any).data.slice(0,10).join(", ")
+                            return "[" + contents + "]";
```

Comment:
if you drop items, you should show "...".

Reply:
Yeah, I'm working on a clean way to do this.